src/index.html

```diff
@@ -5,10 +5,82 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "container">
+      <img src=" images/logo.png" 
+      alt= "moyo_logo" 
+      class = "logo"
+      ></img>
+      <nav class = "main_nav">
```

Comment:
```suggestion
      <nav class="main_nav">
```
в атрибутах не має бути пробілів